### PR TITLE
add to the global cfg

### PR DESCRIPTION
**Pull request by smonero**
add the terraform_admin_mode to the global cfg since we are reading it from the yaml

### CODE REVIEW

**Review by samrabelachew — DISMISSED** [review]

**Comment by smonero** [comment]
now tested in staging

File: server/config/raw/global_cfg.go [hunk 1]
```diff
@@ -26,6 +26,26 @@ type GlobalCfg struct {
...
+	return valid.TerraformAdminMode{
+		Repo: t.Repo,
+		Root: t.Root,
+	}
+}
+
+func (t TerraformAdminMode) Validate() error {
+	return validation.ValidateStruct(&t,
+		validation.Field(&t.Repo),
+		validation.Field(&t.Root),
```
Comment (samrabelachew) [inline server/config/raw/global_cfg.go:47]: did you intend on adding a rule here?

Comment (smonero) [reply server/config/raw/global_cfg.go:47]: I did not. I mean I guess we could but the problem is the string could just be garbage. what are your thoughts?

**Pull request merged**